assets/src/block-templates/high-level-topic/template.js

```diff
@@ -0,0 +1,44 @@
+import gravityFormWithText from '../templates/gravity-form-with-text';
+
+const {__} = wp.i18n;
+
+const template = () => ([
+  ['core/group', {className: 'block'}, [
+    ['planet4-block-templates/page-header', {
+      titlePlaceholder: __('Page header title', 'planet4-blocks-backend'),
+      mediaPosition: 'right',
+    }],
+    ['core/spacer', {height: '64px'}],
+    ['planet4-block-templates/reality-check'],
+    ['planet4-block-templates/side-image-with-text-and-cta', {
+      title: __('The problem', 'planet4-blocks'),
+      alignFull: true,
+    }],
+    ['planet4-block-templates/deep-dive', {
+      title: __('Better understand the issues [deep dive topics]', 'planet4-blocks'),
+      backgroundColor: 'white',
+    }],
+    ['planet4-block-templates/side-image-with-text-and-cta', {
+      title: __('What we do', 'planet4-blocks'),
+      mediaPosition: 'right',
+      alignFull: true,
+    }],
+    ['planet4-block-templates/highlighted-cta', {
+      titlePlaceholder: __('Featured action title', 'planet4-blocks'),
+    }],
+    ['planet4-blocks/covers', {
+      cover_type: 'take-action',
+      title: __('How you can help', 'planet4-blocks'),
+    }],
```

Comment:
Missing an Articles block between the 2 covers I think ?

Reply:
Ah, sorry, my bad 🙈 fixed!